spring-cloud-alibaba-examples/integrated-example/integrated-frontend/src/main/resources/templates/order.html

```diff
@@ -34,7 +34,7 @@
         var userId = $("#userId").val();
         var commodityCode = $("#commodityCode").val();
         $.ajax({
-            url: "http://gateway-service:30010/storage/",
+            url: "http://localhost:30010/storage/",
```

Comment:
Don't modify here, we tell user to set hosts in user 's document. there are same problems below.

Reply:
I forgot to delete it when I was doing some tests.